foladipo/docs-cabinet-cp2
# Feature/148100401/enable searching for documents

#### What does this PR do?

This PR enables a user to search for documents that are either public, his/her own or accessible to all members of his/her role.

#### Description of task to be completed

Enable a user to search for documents.

#### How should this be manually tested?

- Follow the instructions stated in this project's README to set up your local instance of this app.

- Consequently, you should have sign up for a new account.

- Open the side menu and click the `"Search for documents"` option. You will then be taken to search page.

- Search away! Note that you can search based on part or all of a document's title.

#### Any background context you want to provide?

N/A

#### What are the relevant Pivotal Tracker stories?

148100401

#### Screenshots

<img width="1278" alt="screen shot 2017-07-09 at 11 03 54 am" src="https://user-images.githubusercontent.com/26963575/27992891-5e4289a6-6496-11e7-892e-3bfc684bc9bf.png">
